File: app/src/main/java/com/example/getfood/ui/foodmenu/FoodMenuDisplayActivityMvpView.java
package com.example.getfood.ui.foodmenu;

import android.content.Context;

import com.example.getfood.ui.base.BaseView;

public interface FoodMenuDisplayActivityMvpView extends BaseView {

    Context getContext();

    void onPasswordResetEmailSentSuccessfully();

    void onPasswordResetEmailSentFailed(String errMsg);

    void openCart();

    void cantOpenCart();
}

File: app/src/main/java/com/example/getfood/ui/foodmenu/FoodMenuDisplayActivityMvpView.kt
package com.example.getfood.ui.foodmenu

import com.example.getfood.ui.base.BaseView

interface FoodMenuDisplayActivityMvpView : BaseView {

    fun onPasswordResetEmailSentSuccessfully()
    fun onPasswordResetEmailSentFailed(errMsg: String?)
    fun openCart()
    fun cantOpenCart()
}